gatsby-node.js

```diff
@@ -134,7 +215,7 @@ exports.sourceNodes = async () => {
 
 exports.createPages = async ({ actions }) => {
     const { createPage } = actions;
-    const metadata = await stitchClient.callFunction('fetchDocument', [
+    const pageMetadata = await stitchClient.callFunction('fetchDocument', [
```

Comment:
Just curious, why change this var name?

Reply:
In `gatsby_node.js` there is also a global `metadata` var. When playing around I noticed the local scope was overriding this one. In this function I think I can change it back, but would like to distinguish in the new helper.